xla/service/gpu/backend_configs.proto

```diff
@@ -120,18 +120,25 @@ message CudnnfMHABackendConfig {
   // Opaque algorithm number and tuning knobs chosen for this fMHA.
   stream_executor.dnn.AlgorithmProto algorithm = 8;
 
-  // The scaling factor multiplied with the BMM1 result. fmha_scale is 0 if the
+  // The scaling factor multiplied with the BMM1 result. fmha_scale is 1 if the
```

Comment:
Why the change from 0 to 1?

Reply:
i think 0 was a typo before. We by default set the scale to 1 in cudnn backend runner if no scale node is found in HLO graph.